helm/confetti-frontend/templates/deployment.yaml

```diff
@@ -24,17 +24,17 @@ spec:
           image: "quay.io/giantswarm/{{ .Values.image.name }}:{{ .Values.image.tag }}"
           imagePullPolicy: Always
           env:
-          - name: CONFETTI_BACKEND_HOST
+          - name: CONFETTI_PUBLIC_BACKEND_HOST
             value: {{ .Values.backendUrl }}
-          - name: CONFETTI_BACKEND_HOST_SECURE
+          - name: CONFETTI_PUBLIC_BACKEND_HOST_SECURE
             value: {{ .Values.backendUrlSecure | quote }}
           ports:
             - containerPort: {{ .Values.port }}
               name: http
           resources:
             requests:
               cpu: 100m
-              memory: 20Mi
+              memory: 250Mi
```

Comment:
Taking advantage of this PR. This was seen here https://gigantic.slack.com/archives/CQ465CFG8/p1610382503000900